pkg/userop/client.go

```diff
@@ -188,6 +189,20 @@ func NewClient(config ClientConfig) (Client, error) {
 		return nil, fmt.Errorf("failed to connect to the bundler RPC: %w", err)
 	}
 
+	var pmRPC RPCBackend
```

Comment:
I'd rename this since the contraction `pm` is not immediately obvious for people who have less experience with userops. But it's up to debate
```suggestion
	var paymasterRPC RPCBackend
```

Reply:
f53a477c00c669e62d72418df74059acc9d9320c